src/App.vue

```diff
@@ -21,17 +20,10 @@ const i18n = useI18n()
 const dispatcher = { dispatcher: 'App.vue' }
 
 let debouncedOnResize
-const error = computed(() => {
-    if (store.state.ui.errors.size > 0) {
-        return store.state.ui.errors.values().next().value
-    }
-    return null
-})
-const warning = computed(() => {
-    if (store.state.ui.warnings.size > 0) {
-        return store.state.ui.warnings.values().next().value
-    }
-    return null
+const showFeedbackPopup = computed(() => {
+    console.error(store.state.ui.errors)
+    console.error(store.state.ui.warnings)
+    return store.state.ui.errors.size + store.state.ui.warnings.size > 0
```

Comment:
you might want to wrap the + operation with parenthesis, to be sure the greater than won't be interpreted first and end up with a `number + boolean` operation

Reply:
the linter removes the parenthesis automatically, so it's safe to assume the addition has precedence over the check.